Wikipedia/View Controllers/WebView/NativeSectionTitleOverlays/WMFTitleOverlayLabel.h

```diff
@@ -0,0 +1,11 @@
+//  Created by Monte Hurd on 7/30/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import <UIKit/UIKit.h>
+#import "PaddedLabel.h"
+
+@interface WMFTitleOverlayLabel : PaddedLabel
+
+@property (nonatomic) NSNumber* sectionId;
```

Comment:
Seems unnecessary.  Ideally we should treat these headers as we would table view cells. They should mostly be view information - avoiding any model knowledge if possible.